SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs

```diff
@@ -33,11 +33,11 @@ public ProcessDataGateway(ISccvDbContext sccvDbContext, DatabaseContext database
             FilterDefinition<BsonDocument> firstNameFilter;
             FilterDefinition<BsonDocument> lastNameFilter;
 
-            if (!string.IsNullOrWhiteSpace(request.MosaicId))
```

Comment:
this is important for getting records history for the person view, so please don't remove it.

Reply:
Added back in